doma-spring-boot-samples/doma-spring-boot-sample-docker-compose/src/test/resources/application.properties

```diff
@@ -1,12 +0,0 @@
-# Enable Docker Compose in tests
-spring.docker.compose.skip.in-tests=false
```

Comment:
Move `spring.docker.compose.skip.in-tests=false` to src/main/resources/application.properties